content/en/real_user_monitoring/session_replay/mobile/privacy_options.md

```diff
@@ -0,0 +1,269 @@
+---
+title: Mobile Session Replay Privacy Options
+kind: documentation
+description: Describes how to configure privacy options for Mobile Session Replay
+aliases:
+further_reading:
+    - link: '/real_user_monitoring/session_replay/mobile'
+      tag: Documentation
+      text: Mobile Session Replay
+    - link: '/real_user_monitoring/session_replay/mobile/app_performance'
+      tag: Documentation
+      text: How Mobile Session Replay Impacts App Performance
+    - link: '/real_user_monitoring/session_replay/mobile/setup_and_configuration'
+      tag: Documentation
+      text: Setup and Configure Mobile Session Replay
+    - link: '/real_user_monitoring/session_replay/mobile/troubleshooting'
+      tag: Documentation
+      text: Troubleshoot Mobile Session Replay
+    - link: '/real_user_monitoring/session_replay'
+      tag: Documentation
+      text: Session Replay
+---
+
+## Overview
+
+Session Replay provides privacy controls to ensure organizations of any scale do not expose sensitive or personal data. Data is stored on Datadog-managed cloud instances and encrypted at rest.
+
+Default privacy options for Session Replay are designed to protect end user privacy and prevent sensitive organizational information from being collected.
+
+By enabling Mobile Session Replay, you can automatically mask sensitive elements from being recorded through the RUM Mobile SDK. When data is masked, that data is not collected in its original form by Datadog's SDKs and thus is not sent to the backend.
+
+## Configuring masking modes
+
+Using the masking modes below, you can override the default setup on a per-application basis.
+
+### Mask all text elements
+
+By default, the `mask all` setting is enabled for all data. With this setting enabled, all text content on screen is replaced with X's, as shown below.
+
+{{< img src="real_user_monitoring/session_replay/mobile/masking-mode-mask-all.png" alt="What your application screen may resemble when `mask all` is enabled." style="width:60%;">}}
+
+{{< tabs >}}
+{{% tab "Android" %}}
+
+   {{< code-block lang="javascript" filename="build.gradle" disable_copy="false" collapsible="true" >}}
+
+   // mask all text elements
+   val sessionReplayConfig = SessionReplayConfiguration.Builder([sampleRate])
+       .setPrivacy(SessionReplayPrivacy.MASK_ALL)
```

Comment:
@mariusc83 Isn't it `MASK`, `ALLOW` and `MASK_USER_INPUT` now? At least that's [what I see in code](https://github.com/DataDog/dd-sdk-android/blob/a898f208524ec31748afaa4f40247ff73ba70913/features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/SessionReplayPrivacy.kt#L54-L74), but we have different names here in doc.

Reply:
Updated, thanks for the callout.